lib/tasks/foreman_templates_tasks.rake

```diff
@@ -57,6 +57,13 @@ namespace :templates do
     }).purge!
   end
 
+  desc 'Clean data created by this plugin, this will permanently delete the data!'
```

Comment:
how about 'Clean default data seeded by this plugin, this results in permanent deletion!'